cfme/tests/infrastructure/test_advanced_search_vms.py

```diff
@@ -333,3 +341,68 @@ def test_cannot_delete_vm_filter_more_than_once(vm_view):
     vm_view.flash.assert_no_error()
     # Try it second time
     assert not vm_view.entities.search.delete_filter(), 'Delete twice accepted!'
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1506672, 1660460, 1718895])
+def test_create_filter_with_multiple_conditions(appliance, provider, request, running_vms):
+    """
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        caseimportance: high
+        initialEstimate: 1/5h
+        setup:
+            1. Navigate to Compute > Infrastructure > Providers.
+            2. Click on Advanced Search Filter.
+        testSteps:
+            1. Create an expression with multiple types of condition. Eg: arg_1 AND arg_2 OR arg_3
+        expectedResults:
+            1. Expression must be created successfully.
+
+    Bugzilla:
+        1506672
+        1660460
+        1718895
+    """
+    filter_name = fauxfactory.gen_alphanumeric()
+
+    vm1_name, vm2_name = running_vms[0], running_vms[1]
```

Comment:
:+1: I hate py2 unpacking but still we are supporting py2 (:

Reply:
I miss one thing. I'm suspect if no running VM on provider then it will raise IndexError. better think we can skip this test or start at least two vms.

Reply:
true! I missed this... This is an assumption I didn't want to make (that we always have at least two running VMs)

Reply:
please `skip` the test for index error.